tsconfig.json

```
@@ -1,9 +1,22 @@
 {
     "compilerOptions": {
-        "lib": ["es2015"],
+        "lib": [
+            "es2015"
+        ],
         "outFile": "./out/rctrando.js",
         "target": "ES5",
-        "removeComments": true
+        "removeComments": true,
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": false,
+        "noFallthroughCasesInSwitch": true,
+        "noImplicitAny": true,
```

if we temporarily disable this how many compiler errors/warnings do we get? can we get it down to 0?
```suggestion
        "noImplicitAny": false,
```